internal/config/config.go

```diff
@@ -1918,6 +1918,22 @@ func buildSerializer(name string, tbl *ast.Table) (serializers.Serializer, error
 		}
 	}
 
+	if node, ok := tbl.Fields["jmespath_expression"]; ok {
+		if kv, ok := node.(*ast.KeyValue); ok {
+			if str, ok := kv.Value.(*ast.String); ok {
+				c.JmespathExpression = str.Value
+			}
+		}
+	}
+
+	if node, ok := tbl.Fields["tags_prefix"]; ok {
+		if kv, ok := node.(*ast.KeyValue); ok {
+			if str, ok := kv.Value.(*ast.String); ok {
+				c.TagsPrefix = str.Value
+			}
+		}
+	}
```

Comment:
How about adding a fields-prefix the same way? Or alternatively define this as a "prefix for elements not handled otherwise"... Does that make sense?